app/block/[block]/page.tsx

```diff
@@ -410,166 +413,297 @@ export default async function BlockDetailsPage({
-                      The number of cycles used by the prover to generate the
-                      proof.
-                    </MetricInfo>
-                  </MetricLabel>
-                  <MetricValue
-                    title={proving_cycles ? formatNumber(proving_cycles) : ""}
                   >
-                    {proving_cycles ? (
-                      formatNumber(proving_cycles, {
-                        notation: "compact",
-                        compactDisplay: "short",
-                        maximumSignificantDigits: 4,
-                      })
-                    ) : (
-                      <Null />
+                    {proof_status === "proved" && (
+                      <Button
+                        variant="outline"
+                        className={cn(
+                          "aspect-square h-8 w-auto gap-2 self-center text-2xl text-primary",
+                          "disabled:bg-body-secondary/10 sm:max-md:w-40 lg:aspect-auto lg:w-40"
+                        )}
+                        size="icon"
+                        isSecondary={!isComplete}
+                        disabled={!isComplete}
+                      >
+                        <ArrowDown />
+                        <span className="hidden text-nowrap text-xs font-bold sm:block md:hidden lg:block">
+                          Download proof
+                        </span>
+                      </Button>
                     )}
-                  </MetricValue>
-                </MetricBox>
-                <MetricBox
-                  className={cn(
-                    "col-span-2 col-start-3 row-span-1 row-start-3",
-                    "sm:col-span-1 sm:col-start-4 sm:row-span-1 sm:row-start-2",
-                    "md:col-span-1 md:col-start-5 md:row-span-1 md:row-start-1",
-                    "sm:max-md:text-end"
-                  )}
-                >
-                  <MetricLabel className="sm:max-md:justify-end">
-                    Proving cost
-                    <MetricInfo>
-                      The cost of generating this proof.
-                      <br />
-                      (USD)
-                    </MetricInfo>
-                  </MetricLabel>
-                  <MetricValue>
-                    {proving_cost ? (
-                      formatNumber(proving_cost, {
-                        style: "currency",
-                        currency: "USD",
-                      })
-                    ) : (
-                      <Null />
+                    {proof_status === "proving" && (
+                      <Tooltip
+                        content={`${team?.team_name ? team.team_name : "Team"} currently generating proof for this block`}
+                      >
+                        <div
+                          className={cn(
+                            "inline-flex items-center justify-center gap-4 rounded-full border border-solid border-current text-primary [&>svg]:shrink-0",
```

Comment:
We could use the `buttonVariants` function to reuse the generated styles we need here or we could create a `FakeButton` component that does the same.

```tsx
<div className={cn(buttonVariants(...), "override-class")}>
```

Reply:
Fixed these in a deeper branch, will pull this branch in to move things along